apps/test/unit/templates/sectionProgress/progressTables/ProgressTableContentViewTest.jsx

```diff
@@ -124,4 +127,13 @@ describe('ProgressTableContentView', () => {
     expect(FORMATTERS[1].callCount).to.equal(expectedCallCount);
     expect(FORMATTERS[2].callCount).to.equal(expectedCallCount);
   });
+
+  it('uses a fixed column width for empty lessons', () => {
+    const wrapper = setUp({columnWidths: null});
+    const headers = wrapper.find('th');
+    expect(headers.at(0).props().style?.maxWidth).to.be.undefined;
+    expect(headers.at(3).props().style.maxWidth).to.equal(
```

Comment:
Nit: you could have a `maxWidth` set and still have a column with width 0. Either checking `minWidth` instead or also checking `minWidth` would guarantee the column isn't collapsed